libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/map/scalebar/MapboxMapScalebarParams.kt

```diff
@@ -1,101 +0,0 @@
-package com.mapbox.navigation.dropin.map.scalebar
-
-import android.content.Context
-import com.mapbox.navigation.base.formatter.UnitType
-import com.mapbox.navigation.base.internal.extensions.LocaleEx.getUnitTypeForLocale
-import com.mapbox.navigation.base.internal.extensions.inferDeviceLocale
-
-/**
- * Params describing the map scalebar config.
- *
- * @param enabled true if the scalebar should be shown on the map, false otherwise
- * @param isMetricUnits true if the scale bar is using metric system,
- *  false if the scale bar is using imperial units.
- */
-class MapboxMapScalebarParams private constructor(
```

Comment:
Why did you remove this file?

Reply:
We removed it's usage here https://github.com/mapbox/mapbox-navigation-android/pull/6523 and forgot to delete `MapboxMapScalebarParams`.